src/components/common/MessageBox/index.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import clsx from "clsx";
+import PropTypes from "prop-types";
+import { withStyles } from "@material-ui/styles";
+
+import { useStyles } from "./styles";
+
+const backgroundColor = {
+  error: "error",
+  success: "success",
+};
+
+const MessageBox = ({ classes, errorMsg, type }) => {
+  if (errorMsg) {
+    return <p className={clsx(classes.messageBox, classes[backgroundColor[type]])}>{errorMsg}</p>;
+  }
+  return null;
+};
+
+MessageBox.propTypes = {
+  type: PropTypes.oneOf(["error", "success"]),
+  showErr: PropTypes.oneOfType([PropTypes.bool, PropTypes.string]),
+  errorMsg: PropTypes.string,
```

Comment:
I would go with just `type` and `message` and based on the type I would pick appropriate `styling`. `message` should be a required prop.

Reply:
[done](https://github.com/Vivek205/refactored-dapp/pull/22/commits/39c50b77dc05ab784e7199ff68395eb7a16ceba4)